xDSCResourceDesigner.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.1.1.2'
+ModuleVersion = '1.1.1.3'
```

Comment:
No problem! Our Contribution doc has changed about 10 times in the last 24 hours, so we're not expecting 100% pull request perfection yet. Now that we're live, we've got plenty of time to get there.